### PR TITLE
Choreform update

### LINKED ISSUE #5: User Story: As a user I should be able to choose the time needed for the task and the frequency needed to be done so that I can be assigned an equal amount of chores as other family members.

### PR DESCRIPTION
**Pull request by Kanderson58**
# Description

- Get choreform fully updated
- Query data from BE to display
- Mutate data with graphQL to POST
- Fix minor CSS errors

## Type Of Change

- [ ]  Bug fix
- [ ]  New feature
- [x]  Refactoring
- [ ]  Testing

# How Has This Been Tested?

- [x]  local host
- [ ]  dev tools
- [ ]  cypress

# Related Issues:

- [x]  closes  #5 

# Checklist:

- [x]  My code follows the Turing's guidelines of this project
- [x]  I have performed a self-review of my own code
- [x]  No console error messages

### CODE REVIEW

File: src/Components/ChoreForm/ChoreForm.js
Comment (LeftyLincoln) [inline src/Components/ChoreForm/ChoreForm.js:1]: We are getting the choreData!

File: src/Components/HouseForm/HouseForm.js
Comment (LeftyLincoln) [inline src/Components/HouseForm/HouseForm.js:1]: all hail onCompleted!

File: src/queries.js
Comment (LeftyLincoln) [inline src/queries.js:1]: Crushing graphql!

**Review by LeftyLincoln — APPROVED** [review]
LGTM

File: src/Components/ChoreCard/ChoreCard.css [hunk 1]
```diff
@@ -7,4 +7,8 @@
   color: aliceblue;
   text-shadow: 2px 2px 5px black;
   font-size: larger;
+}
+
+h2 {
+  text-transform: capitalize;
```
Comment (qrispi) [inline src/Components/ChoreCard/ChoreCard.css:13]: Nice error handling!

File: src/Components/ChoreForm/ChoreForm.js [hunk 2]
```diff
@@ -40,27 +52,35 @@ const ChoreForm = ({id, email}) => {
...
       }
-      console.log(data);
-      clearForm();
+      addChore({variables: { input }});
     }
   }
 
   const clearForm = () => {
-    setInput({
+    setChoreInput({
```
Comment (qrispi) [inline src/Components/ChoreForm/ChoreForm.js:68]: Thanks for sharing that search and replace tip!

**Review by qrispi — APPROVED** [review]
LGTM! Great work today!

**Pull request merged**